src/commands/dev/trace.js

```diff
@@ -1,8 +1,11 @@
+const { flags } = require('@oclif/command')
 const Command = require('../../utils/command')
 const { runProcess } = require('../../utils/traffic-mesh')
 
 class TraceCommand extends Command {
   async run() {
+    this.parse(TraceCommand)
```

Comment:
This is done to validate the argument and print an error consistent with the other CLI commands (otherwise the error comes directly from the Traffic Mesh agent).